### DISCUSSION TITLE
有没有可能计划支持windows11

### DISCUSSION THREAD

**Discussion by ghost** [fcitx/fcitx5#1104] [Ideas]
rime不论是在Win还是android都不能开箱即用，对于普罗大众是一个很高的门槛，但是fcitx很易用**（特别是在Android）**。目前大厂的输入法表明没有广告了，但是卖用户数据。这无疑有害隐私保护，如果有Windows支持，我就能向朋友推荐fcitx了。

**Comment by directentis1** [comment]
你应该推荐你的朋友使用Linux，会更实用 😀

**Comment by xingwozhonghua126** [comment]
小狼毫就可以，windows自带的也没看到推广告。

**Reply by ghost** [reply]
> 小狼毫就可以，windows自带的也没看到推广告。

小狼毫配置麻烦，之前折腾过安卓版。Win的也折腾，一点都不现代化，图标也不好看
